mlrun/serving/routers.py

```diff
@@ -1042,7 +1045,7 @@ def _init_endpoint_record(
         versioned_model_name = f"{voting_ensemble.name}:latest"
 
     # Generating model endpoint ID based on function uri and model version
-    endpoint_uid = mlrun.utils.model_monitoring.create_model_endpoint_id(
+    endpoint_uid = create_model_endpoint_id(
```

Comment:
I think the function name should be change, uid and id has a different meaning and it can confuse

Reply:
Changed to `create_model_endpoint_uid`.